lib/python-beta/src/bailo/helper/model.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import annotations
+
+import os
+from typing import Any
+
+from bailo.core import Client, ModelVisibility
+from bailo.helper.utils import retrieve_model
+
+
+class Model:
+    def __init__(
+        self,
+        client: Client,
+        name: str,
+        description: str,
+        visibility: ModelVisibility | None = ModelVisibility.Public,
+        model_card: dict[str, Any] | None = None
+    ):
+        self.client = client
+        self.name, self.description, self.visibility = name, description, visibility
+        self.model_id = None
+        self.model_version = None
+        self.model_card_version = None
+        self.model_card_schema = 'minimal-general-v10-beta'
+        self.__model_card_original = model_card
+        self.__model_card = model_card
+        self.__local_dir = f"./temp_{name.strip(' ')}"  # property tbc
```

Comment:
Using the `tempfile` module to get a temporary store will likely work better.  In a lot of cases (e.g. when deployed to a Docker container) the local directory may not be writable.

https://docs.python.org/3/library/tempfile.html